pkg/kube/inject/webhook.go

```diff
@@ -500,6 +526,10 @@ func createPatch(pod *corev1.Pod, prevStatus *SidecarInjectionStatus, annotation
 
 	patch = append(patch, updateAnnotation(pod.Annotations, annotations)...)
 
+	if pod.Labels[MTLSReadyLabelName] == "" {
```

Comment:
@incfly the overwrite case is handled here - I never call `updateLabels()` if mtlsReady is set to something already

Reply:
okay, then should this be renamed to `addLabel`? Since `update` can be interpreted as override something existing.